### DISCUSSION TITLE
How to rebind equal (=) key?

### DISCUSSION THREAD

**Discussion by kirusfg** [helix-editor/helix#4786] [Q&A] [answered]
Title.

```toml
[keys.normal]
= = "reflow 80"
```

clearly would not work. What name does the equal sign have as a key?

**Comment by CptPotato** [comment]
You can use quotes for characters that aren't directly allowed in toml: `"=" = "reflow 80"`

**Reply by kirusfg** [reply]
I tried that as well before posting the question, and it did not work. What I did not realise is that the problem is caused by the `reflow 80` part. For example,
```toml
"=" = "keep_primary_selection"
```
works, while
```toml
"=" = "reflow 80"
```
causes an error:
```
Bad config: data did not match any variant of untagged enum KeyTrie for key `keys.normal` at line 27 column 1
```

Turns out it must be
```toml
"=" = ":reflow 80"
```

**Comment by kirusfg** [comment, marked as answer]
```toml
"=" = ":reflow 80"
```
